app/assets/javascripts/admin/util.js.erb

```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
   $('.datetimepicker').datetimepicker({
-    dateFormat: Spree.translations.date_picker,
+    dateFormat: "yy-mm-dd",
```

Comment:
I thought it would be best to just fix it, as any other formats can break the datepicker. If it's taking translations it could use the wrong format in some situations, no?